frontend/app/utils/ReferralData.scala

```diff
@@ -6,15 +6,16 @@ case class ReferralData(campaignCode: Option[String], url: Option[String], pagev
 
 object ReferralData {
 
-  val CampaignCodeKey = "mem_campaign_code"
+  val CampaignCodeKey =  "mem_campaign_code"
   val UrlKey = "gu_mem_ref_url"
   val PageviewIdKey = "gu_refpvid"
 
   def makeCookies(implicit request: RequestHeader): Seq[Cookie] = {
     val refUrl = request.headers.get("referer").map(Cookie(ReferralData.UrlKey, _))
     val refPvid = request.getQueryString("REFPVID").map(Cookie(ReferralData.PageviewIdKey, _))
+    val campaignCode = request.getQueryString("INTCMP").map(Cookie(ReferralData.CampaignCodeKey, _))
```

Comment:
Another thought is that we don't exclusively use `INTCMP` for campaign codes, we sometimes use `CMP` too